blog/ga-maint-for-neon.adoc

```diff
@@ -0,0 +1,227 @@
+= JBoss Tools and Red Hat Developer Studio Maintenance Release for Eclipse Neon
+:page-layout: blog
+:page-author: jeffmaury
+:page-tags: [release, jbosstools, devstudio, jbosscentral]
+:page-date: 2016-09-27
+
+link:/downloads/jbosstools/neon/4.4.1.Final.html[JBoss Tools 4.4.1] and link:/downloads/devstudio/neon/10.1.0.GA.html[Red Hat JBoss Developer Studio 10.1] for Eclipse Neon are here waiting for you. Check it out!
+
+image::/blog/images/devstudio10.png[]
+
+== Installation
+
+JBoss Developer Studio comes with everything pre-bundled in its installer. Simply download it from our https://www.jboss.org/products/devstudio.html[JBoss Products page] and run it like this:
+ 
+    java -jar jboss-devstudio-<installername>.jar
+
+JBoss Tools or Bring-Your-Own-Eclipse (BYOE) JBoss Developer Studio require a bit more:
+
+This release requires at least Eclipse 4.6 (Neon) but we recommend
+using the latest http://www.eclipse.org/downloads/packages/eclipse-ide-java-ee-developers/neonr[Eclipse 4.6 Neon JEE Bundle] since then you get most of the dependencies preinstalled. 
+
+Once you have installed Eclipse, you can either find us on the Eclipse Marketplace under "JBoss Tools" or "Red Hat JBoss Developer Studio".
+
+For JBoss Tools, you can also use our update site directly.
+
+    http://download.jboss.org/jbosstools/neon/stable/updates/
+
+== What is new? 
+
+Our main focus for this release was improvements for container based development and bug fixing.
+
+=== Improved OpenShift 3 and Docker Tools
+
+We continue to work on providing better experience for container based development in JBoss Tools and Developer Studio. Let's go through a few interesting updates here and you can find more details on link:/documentation/whatsnew/jbosstools/4.4.0.Final.html[the What's New page].
```

Comment:
We continue to work on providing better experience for container based development in JBoss Tools and Developer Studio. Let's go through the interesting updates here.